src/llvm/lib_llvm_ext.cr

```diff
@@ -86,8 +94,17 @@ lib LibLLVMExt
                                                                       align_in_bits : UInt64,
                                                                       name : LibC::Char*) : Metadata
 
+{% if LibLLVM::IS_38 %}
```

Comment:
This should be the other way around: if IS_35 or IS_36, because we also support 3.9 now, and I expect the functions to be available in 4.x too.